src/source.scss

```diff
@@ -1,4 +1,4 @@
-.platform-win {
+.platform-win, .platform-linux {
```

Comment:
```suggestion
:is(.platform-win, .platform-linux) {
```

Reply:
Good suggestion. Also, my Discord I've just installed today and it uses a custom bar now similar to Windows, see: 
<img width="671" height="160" alt="image" src="https://github.com/user-attachments/assets/ddfa00fd-c03e-4c34-b85e-bb048d03c7aa" />